src/aap_eda/settings/post_load.py

```diff
@@ -244,26 +244,66 @@ def _get_spectacular_settings(settings: Dynaconf) -> dict:
     }
 
 
+STREAM_HANDLER = "logging.StreamHandler"
+
+
 def _get_logging_setup(settings: Dynaconf) -> dict:
     return {
         "version": 1,
         "disable_existing_loggers": False,
+        "filters": {
+            "activation_id_filter": {
+                "()": "aap_eda.utils.log_tracking_id_filter.LogTrackingIdFilter",  # noqa: E501
+            },
+        },
         "formatters": {
             "simple": {
-                "format": "{asctime} {name} {levelname:<8} {message}",
+                "format": "{asctime} {levelname:<8} {name} {message}",
+                "style": "{",
+            },
+            "simple_with_request": {
+                "format": "{asctime} {levelname:<8} [rid:{request_id}] "
```

Comment:
could we do `x-request-id` may help the poor souls who have to dig through SOS report know what it is cc @chrismeyersfsu

Reply:
I was initially thinking it's better to abbreviate it since these log lines can already get quite long, and then we ensure we document the abbreviation in our docs. But perhaps it is better to use the few extra characters and make it immediately clear in the logs. Wdyt @kdelee @hsong-rh @mkanoor ?

Reply:
@ttuffin Agreed, the log lines are pretty long. We can change it in the future when using JSON format.